model/registry/test.yaml

```diff
@@ -0,0 +1,37 @@
+groups:
+  - id: registry.test.suite
+    prefix: test
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to tests.
+    attributes:
+      - id: suite.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the test suite.
+        examples: ['TestSuite1']
+      - id: suite.id
+        type: string
+        stability: experimental
+        brief: >
+          The id of the test suite.
+        examples: ['123408']
+      - id: suite.case.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the test case.
+        examples: ['TestCase1']
+      - id: suite.case.id
+        type: string
+        stability: experimental
+        brief: >
+          The id of the test case.
+        examples: ['123445']
+      - id: suite.case.status
+        type: string
```

Comment:
should it be an enum?

Reply:
I do think this would be good as a enum, `success | failure | skipped`. Thoughts?

Reply:
Have updated `run.status` and `result.status` enums. They do not define all of those values, but they do allow custom ones to be set. The `test.yaml` has changed a bit since the last review so feel free to give another look.